tests/clickhouse-test

```diff
@@ -573,27 +574,6 @@ def get_localzone():
     return os.getenv("TZ", "/".join(os.readlink("/etc/localtime").split("/")[-2:]))
 
 
-def supports_io_uring():
-    return not subprocess.call(
-        [
-            args.binary,
-            "-q",
-            "select * from file('/dev/null', 'LineAsString')",
-            "--storage_file_read_method",
-            "io_uring",
-        ],
-        stdout=subprocess.DEVNULL,
-        stderr=subprocess.DEVNULL,
-    )
-
-
-def get_local_filesystem_methods():
```

Comment:
Instead of reverting this, maybe it is better to add a test with **explicit** `io_uring` method requested?